llvm/lib/MC/MCCASObjectV1.cpp

```diff
@@ -1920,9 +2097,11 @@ static void partitionCUDie(
 }
 
 Expected<InMemoryCASDWARFObject::PartitionedDebugInfoSection>
-InMemoryCASDWARFObject::partitionCUData(ArrayRef<char> DebugInfoData,
+InMemoryCASDWARFObject::partitionCUData(MCCASBuilder &Builder,
+                                        ArrayRef<char> DebugInfoData,
                                         uint64_t AbbrevOffset,
-                                        uint64_t CUOffset, DWARFContext *Ctx) {
+                                        uint64_t CUOffset, DWARFContext *Ctx,
+                                        ArrayRef<DebugStrRef> DebugStringRefs) {
```

Comment:
Is `DebugStringRefs` used anywhere in this function?

Reply:
You are right, it is not used, I changed the code to address that